package.json

```diff
@@ -40,6 +40,9 @@
         "@types/strip-json-comments",
         "strip-json-comments"
       ]
+    },
+    "patchedDependencies": {
+      "jest-file-snapshot@0.5.0": "patches/jest-file-snapshot@0.5.0.patch"
```

Comment:
This is due to https://github.com/satya164/jest-file-snapshot/pull/23 (which I only found after I had patched it locally myself).